jetstream/tests/integration/test_analysis_integration.py

```diff
@@ -322,17 +375,19 @@ def test_metrics_with_exposure(
         """
         )
 
+        a_value = "0" if analysis_unit == AnalysisUnit.PROFILE_GROUP.value else "a"
+        b_value = "0" if analysis_unit == AnalysisUnit.PROFILE_GROUP.value else "b"
```

Comment:
This line is repeated across a lot of tests. How about refactoring into a helper function, something like:

```python
def analysis_unit_value(letter: str, analysis_unit: str) -> str:
    return "0" if analysis_unit == AnalysisUnit.PROFILE_GROUP.value else letter
```

Which can then just be used in the f-string: 
```python
"analysis_id": f"bbb{analysis_unit_value('b', analysis_unit)}"
```

Reply:
Yep, good idea!